livingdoc-converters/src/main/kotlin/org/livingdoc/converters/collection/AbstractCollectionConverter.kt

```diff
@@ -0,0 +1,57 @@
+package org.livingdoc.converters.collection
+
+import org.livingdoc.api.conversion.ConversionException
+import org.livingdoc.api.conversion.TypeConverter
+import org.livingdoc.converters.TypeConverters
+import java.lang.reflect.*
+
+abstract class AbstractCollectionConverter<T : Collection<Any>> : TypeConverter<T> {
+
+    val defaultSeparator = ","
+
+    @Throws(ConversionException::class)
+    override fun convert(value: String, element: AnnotatedElement?, documentClass: Class<*>?): T {
+        val tokenized: List<String> = tokenize(value)
+        val paramsConverted = convertIntoParameterType(element, documentClass, tokenized)
+
+        return convertToTarget(paramsConverted)
+    }
+
+    private fun tokenize(value: String): List<String> {
+        return value.split(delimiters = defaultSeparator).map { it.trim() }
+    }
+
+    private fun convertIntoParameterType(element: AnnotatedElement?, documentClass: Class<*>?, tokenized: List<String>): List<Any> {
+
+        val paramTypeConverter = when (element) {
+            is Field -> {
+                converterFromField(element, documentClass)
+            }
+            is Parameter -> {
+                converterFromParameter(element, documentClass)
+            }
+            else -> null
+        } ?: throw IllegalStateException()
+
+        return tokenized.map { paramTypeConverter.convert(it, element, documentClass) }.toList()
+    }
+
+    private fun converterFromParameter(parameter: Parameter, documentClass: Class<*>?): TypeConverter<*>? {
```

Comment:
Method should be named something like `findTypeConverter(..)`. Same for `Field` related method below.